js/blockchaincom.js

```diff
@@ -839,12 +839,15 @@ module.exports = class blockchaincom extends Exchange {
     }
 
     async withdraw (code, amount, address, tag = undefined, params = {}) {
```

Comment:
I would add some docs here explaining what beneficiary is.

`Blockchain does not support to provide addresses for withdrawals. Provide the whitelist id instead through params object. Whitelist id can be obtained through whitelist endpoint: https://api.blockchain.info/v3/#/payments/getWithdrawals`